src/components/Attachments/AttachmentCarousel/extractAttachmentsFromReport.ts

```diff
@@ -76,9 +76,9 @@ function extractAttachmentsFromReport(parentReportAction?: OnyxEntry<ReportActio
             return;
         }
 
-        const decision = action?.message?.[0]?.moderationDecision?.decision;
+        const decision = ReportActionsUtils.getReportActionMessage(action)?.moderationDecision?.decision;
```

Comment:
Please introduce a local variable, so we don't repeat ourselves and so the lines don't get extremely long.

Please do that in other spots of the PR, wherever it's easy to do.